.github/workflows/check-packages.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           node-version: 14.18.1
           cache: 'yarn'
 
-      - uses: tj-actions/changed-files@v32
+      - uses: tj-actions/changed-files@v34
```

Comment:
hey thanks for this ! we are using this action in another workflows as well. would you mind to update all of those ? cheers